### ISSUE TITLE
calAsyncUtils.jsm missing

### ISSUE THREAD

**Issue by hans109h** [kewisch/gdata-provider#416]
### Describe the problem and steps to reproduce it:
Open Thunderbird 99.0b2, open calendar error "Missing chrome or resource URL: resource:///modules/calendar/utils/calAsyncUtils.jsm" occurs and no events from synced google calendar appear.  Events are able to be added to the google calendar, and can be viewed using other software/web but do not appear in Thunderbird.

### What happened?
Google calendar information is completely missing

### What did you expect to happen?
Google Calendar information to display in Thunderbird.

### Anything else we should know?

```
1648130175997	addons.xpi	WARN	Checking /opt/thunderbird/distribution/extensions for addons
Calendar: [CalTimezoneService] Loading resource:///res/zones.json CalTimezoneService.jsm:86
Calendar: [CalTimezoneService] Timezones version 2.2021e loaded CalTimezoneService.jsm:132
Calendar: [CalMetronome] Scheduling one-off event in 43416ms CalMetronome.jsm:103
This page is in Quirks Mode. Page layout may be impacted. For Standards Mode use “<!DOCTYPE html>”. hiddenWindow.html
Calendar: [calGoogleSessionManager] Creating session ***@gmail.com gdataSession.jsm:88
Calendar: [calGoogleSessionManager] Reusing session ***@gmail.com 4 gdataSession.jsm:86
Calendar: [calCachedCalendar] Performing playback operation add on 0 items to ***@gmail.com calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation add on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation add on 0 items to Holidays in United States calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation add on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation modify on 0 items to ***@gmail.com calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation add on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation modify on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation modify on 0 items to Holidays in United States calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation modify on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation delete on 0 items to ***@gmail.com calCachedCalendar.js:552
Calendar: [calCachedCalendar] Doing changelog based sync for calendar googleapi://***@gmail.com/?calendar=***%40gmail.com calCachedCalendar.js:300
Calendar: [calGoogleSession] Token expired 1648130177 seconds ago, resetting gdataSession.jsm:388
Calendar: [calGoogleCalendar] Logging in session ***@gmail.com gdataSession.jsm:266
Calendar: [calGoogleCalendar] No access token for ***@gmail.com, refreshing token gdataSession.jsm:347
Calendar: [calGoogleSession] Adding item https://www.googleapis.com/calendar/v3/users/me/calendarList/***%40gmail.com to queue gdataSession.jsm:407
Calendar: [calCachedCalendar] Performing playback operation modify on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Performing playback operation delete on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Doing changelog based sync for calendar googleapi://***@gmail.com/?calendar=7j69sims35cgfi8u6l0937p6ss%40group.calendar.google.com calCachedCalendar.js:300
Calendar: [calGoogleSession] Adding item https://www.googleapis.com/calendar/v3/users/me/calendarList/7j69sims35cgfi8u6l0937p6ss%40group.calendar.google.com to queue gdataSession.jsm:407
Failed to load module resource:///modules/calendar/utils/calAsyncUtils.jsm. XPCOMUtils.jsm:240
Calendar: [calCachedCalendar] Performing playback operation delete on 0 items to Holidays in United States calCachedCalendar.js:552
Calendar: [calCachedCalendar] Doing changelog based sync for calendar googleapi://***@gmail.com/?calendar=en.usa%23holiday%40group.v.calendar.google.com calCachedCalendar.js:300
Calendar: [calGoogleSession] Adding item https://www.googleapis.com/calendar/v3/users/me/calendarList/en.usa%23holiday%40group.v.calendar.google.com to queue gdataSession.jsm:407
Calendar: [calCachedCalendar] Performing playback operation delete on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Doing changelog based sync for calendar googleapi://***@gmail.com/?calendar=a89gmc0t9ab9o77lvsqbo6kmjs%40group.calendar.google.com calCachedCalendar.js:300
Calendar: [calGoogleSession] Adding item https://www.googleapis.com/calendar/v3/users/me/calendarList/a89gmc0t9ab9o77lvsqbo6kmjs%40group.calendar.google.com to queue gdataSession.jsm:407
Calendar: [calCachedCalendar] Performing playback operation delete on 0 items to *** calCachedCalendar.js:552
Calendar: [calCachedCalendar] Doing changelog based sync for calendar googleapi://***@gmail.com/?calendar=oqmm3jd2s98c52vh3grsmkcbf8%40group.calendar.google.com calCachedCalendar.js:300
Calendar: [calGoogleSession] Adding item https://www.googleapis.com/calendar/v3/users/me/calendarList/oqmm3jd2s98c52vh3grsmkcbf8%40group.calendar.google.com to queue gdataSession.jsm:407
NS_ERROR_FILE_NOT_FOUND XPCOMUtils.jsm:234
TypeError: cal.async is undefined 4 gdataUtils.jsm:1028:3
Trying to load /opt/thunderbird/libotr.so OTRLib.jsm:64:11
Successfully loaded OTR library /opt/thunderbird/libotr.so OTRLib.jsm:72:13
Successfully loaded OpenPGP library librnp.so version 0.16+git20220124.f06439f7.MZLA from /opt/thunderbird/librnp.so RNPLib.jsm:100:15
Assert failed: Calendar manager initialised calendars before loadCalendarComponent ran on the first 3pane window. This should not happen.
 calUtils.jsm:121
Found 0 public keys and 0 secret keys (0 protected, 0 unprotected) RNPLib.jsm:302:15
services.settings: Failed to load last_modified.json: TypeError: NetworkError when attempting to fetch resource. Utils.jsm:248
Calendar: Start loading of locale dependent preference default values... calendar-chrome-startup.js:328
Calendar: Loading of locale sensitive preference default values completed. calendar-chrome-startup.js:348
services.settings: thunderbird/hijack-blocklists has signature disabled RemoteSettingsClient.jsm:1027
Calendar: TypeError: this._stream is undefined
    Symbol.asyncIterator resource:///modules/calendar/utils/calIteratorUtils.jsm:263
    start resource:///modules/CalReadableStreamFactory.jsm:274
calendar-invitations-manager.js:125
tb.account.size_on_disk - Truncating float/double number. 2
tb.account.size_on_disk - Trying to set an unsigned scalar to a negative number. 2
Calendar: [calGoogleCalendar] Successfully acquired a new OAuth token for ***@gmail.com gdataSession.jsm:270
Calendar: [calGoogleRequest] Requesting GET https://www.googleapis.com/calendar/v3/users/me/calendarList/***%40gmail.com gdataRequest.jsm:212
Calendar: [calGoogleRequest] Requesting GET https://www.googleapis.com/calendar/v3/users/me/calendarList/7j69sims35cgfi8u6l0937p6ss%40group.calendar.google.com gdataRequest.jsm:212
Calendar: [calGoogleRequest] Requesting GET https://www.googleapis.com/calendar/v3/users/me/calendarList/en.usa%23holiday%40group.v.calendar.google.com gdataRequest.jsm:212
Calendar: [calGoogleRequest] Requesting GET https://www.googleapis.com/calendar/v3/users/me/calendarList/a89gmc0t9ab9o77lvsqbo6kmjs%40group.calendar.google.com gdataRequest.jsm:212
Calendar: [calGoogleRequest] Requesting GET https://www.googleapis.com/calendar/v3/users/me/calendarList/oqmm3jd2s98c52vh3grsmkcbf8%40group.calendar.google.com gdataRequest.jsm:212
Calendar: [calGoogleCalendar] Request GET https://www.googleapis.com/calendar/v3/users/me/calendarList/oqmm3jd2s98c52vh3grsmkcbf8%40group.calendar.google.com responded with HTTP 200 gdataRequest.jsm:389
Calendar: [calGoogleCalendar] Request GET https://www.googleapis.com/calendar/v3/users/me/calendarList/***%40gmail.com responded with HTTP 200 gdataRequest.jsm:389
Calendar: [calGoogleCalendar] Request GET https://www.googleapis.com/calendar/v3/users/me/calendarList/en.usa%23holiday%40group.v.calendar.google.com responded with HTTP 200 gdataRequest.jsm:389
Calendar: [calGoogleCalendar] Request GET https://www.googleapis.com/calendar/v3/users/me/calendarList/7j69sims35cgfi8u6l0937p6ss%40group.calendar.google.com responded with HTTP 200 gdataRequest.jsm:389
Calendar: [calGoogleCalendar] Request GET https://www.googleapis.com/calendar/v3/users/me/calendarList/a89gmc0t9ab9o77lvsqbo6kmjs%40group.calendar.google.com responded with HTTP 200 gdataRequest.jsm:389

```

**Comment by hans109h** [comment]
Related to: https://github.com/kewisch/gdata-provider/discussions/407

**Comment by 9strands** [comment]
Doing a bit of research, appears that calAsyncUtils.jsm was removed here: https://github.com/mozilla/releases-comm-central/commit/7570d7a91651102ae0edc8d770404e7c13179039

**Comment by arlenk** [comment]
Same issue here.  It looks like offline storage methods are all async now so cal.async isn't necessary any more.

https://bugzilla.mozilla.org/show_bug.cgi?id=1693873

**Comment by kewisch** [comment]
I've fixed this in 92bc1d80e4e9499df0253a14bbbe18cce7f3ed1a
